### ISSUE TITLE
Making a prediction from ONNX model in ML.net

### ISSUE THREAD

**Issue by Varusnguyen** [dotnet/machinelearning#3562]
### System information

- **OS version: window 10
- Ml.Net verison: 0.11

### Issue
Hi, everyone. I am a newbie of using ML.net to implement my classification problem. I already trained a model from CIFA 10 datasets based on Keras library and that model has been transferred to ONNX model. My goal is to display the image's prediction task on WindowForm, so I am going to use ML.net integrated with ONNX model, but right now I am getting stuck on how to use the ONNX model on ML.net and how to define the label and get the prediction result by the model. Do you have any suggestion to me? Thank you a lot in advance.
### Source code / logs

Please paste or attach the code or logs or traces that would be helpful to diagnose the issue you are reporting.

**Comment by PeterPann23** [comment]
Hi Varusnguyen 

Nice to see you, have a look at [the onnox sample](https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx)

Also the tests in the project are a good start.

**Comment by usmankaiiz** [comment]
Can someone guide me, how to pass video frames instead of pictures which are stored in folders? in the pipeline?

var pipeline = mlContext.Transforms.LoadImages(outputColumnName: "image", imageFolder: imagesFolder,

**Comment by PeterPann23** [comment]
Hi, Not sure, never did that, however, how will you correctly label them so the model learns... 
When I Look on the www I get this [method to capture frames from a video](https://stackoverflow.com/questions/35380868/extract-frames-from-video-c-sharp). might not work for you but if it helps you than glad to help.

**Comment by yaeldekel** [comment]
Hi @Varusnguyen, thank you for trying out ML.NET.
Issue #3460 discusses how to pass Bitmaps instead of pictures stored in folders, you may find it useful.

**Comment by Varusnguyen** [comment]
Thanks for many useful suggestions, it helped me a lot to gain knowledge about ONNX and ML.net.  Btw, I also found out a helpful material of using C# to display the result from ONNX model, you can read to get more details.
https://github.com/microsoft/onnxruntime

**Comment by CESARDELATORRE** [comment]
@usmankaiiz - This might interest you. Since ML.NET 1.1 we can also very easily load in-memory images into the pipeline targeting an ONNX or TensorFlow models, for instance.

For ONNX, this example is using in-memory images instead of images coming from files on the hard drive:

https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/end-to-end-apps/DeepLearning_ObjectDetection_Onnx

**Comment by nfnpmc** [comment]
Link is dead!

**Comment by CESARDELATORRE** [comment]
It was moved here:
https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/end-to-end-apps/ObjectDetection-Onnx
